gpu_ocean/demos/testCasesDemos/run_benchmark.py

```diff
@@ -38,6 +38,8 @@
 parser.add_argument('--iterations', type=int, default=1)
 parser.add_argument('--simulator', type=str)
 parser.add_argument('--output', type=str, default=None)
+parser.add_argument("--new_fbl", type=int, default=0)
```

Comment:
I think old-fbl should be removed, thus not requiring this

Reply:
old-fbl is required in order to benchmark the original "untuned FBL"

Reply:
Isn't that a git checkout away? That's what we do with the other ones, right?